tests/headers/host_test.go

```diff
@@ -111,3 +104,51 @@ func TestHostHeaderMultiple(t *testing.T) {
 		t.Errorf("status code got: %q want: %q", got, want)
 	}
 }
+
+func TestAbsoluteFormURLInvalidSchema(t *testing.T) {
+	// When sending a request using the absolute
+	// form as the request target, any schema is currently
+	// accepted.
+	//
+	// The desired behavior would instead be that only
```

Comment:
"to only accept http or https as schemas and to respond with a 400 (Bad Request) when the server receives any other schema"